.github/workflows/ci2.yml

```diff
@@ -3,8 +3,9 @@ name: Build
 on:
   pull_request:
 
+# Unique to this workflow and PR, or SHA if not a PR
 concurrency:
-  group: ci2-${{ github.head_ref }}
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}
```

Comment:
We only run this on pull_request so it's not an issue, but I updated for consistency. If we only use `head_ref` for uniqueness, and if this was ran on landing, then e.g. if we landed two commits to master back-to-back, the second would cancel the workflow for the first. Using `sha` for non pull requests means we maintain uniqueness for landing runs